libsimpleservo build is broken

I encountered this while trying to build for Android:

```
error[E0004]: non-exhaustive patterns: `EmbedderMsg::ReadyToPresent` not covered
   --> ports/libsimpleservo/api/src/lib.rs:623:19
    |
623 |             match event {
    |                   ^^^^^ pattern `EmbedderMsg::ReadyToPresent` not covered
    |
```

cc @delan @atbrakhi

--- a/ports/libsimpleservo/api/src/lib.rs
+++ b/ports/libsimpleservo/api/src/lib.rs
@@ -802,6 +802,7 @@ impl ServoGlue {
                 EmbedderMsg::NewFavicon(..) |
                 EmbedderMsg::HeadParsed |
                 EmbedderMsg::SetFullscreenState(..) |
+                EmbedderMsg::ReadyToPresent |
                 EmbedderMsg::ReportProfile(..) => {},
             }
         }

